### DISCUSSION TITLE
VMWARE OS Install

### DISCUSSION THREAD

**Discussion by larson167** [Security-Onion-Solutions/securityonion#9118] [General]
Good morning,

I am trying to install and and set up SO. I am currently using cent OS to do this on VMWARE, I have 3 network adapters one is set to Nat and the other two are custom. I am watching a video on how to set this up but when I go to choose my adapter that is set to NAT like in the video it shows me ens32 is showing down. any help will be appreciated .
![IMG_6042](https://user-images.githubusercontent.com/104645429/201366221-66219328-9232-4cc1-95cb-d38ae1b412d6.jpg)
![IMG_6043](https://user-images.githubusercontent.com/104645429/201366253-d70e16b9-91c5-4768-ad26-b8db0ece457a.jpg)

**Comment by dougburks** [comment]
In VMware Virtual Machine Settings, make sure that `Connected` is checked under the Network Adapter device status.
